sdk/include/asm/asm.inc

```diff
@@ -184,8 +198,12 @@ ljmp16 MACRO segment, offset
     DW segment
 ENDM
 
+data32prefix MACRO
+    DB 066h
+ENDM
+
```

Comment:
```suggestion
.macro data32, opcode:vararg
    .byte 0x66
    opcode
.endm
```
This overwrites the built in `data32` to stop GAS from complaining about a redundant prefix.